package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "stream",
-  "version": "0.0.2",
+  "version": "0.0.3",
```

Comment:
please revert the version upgrade, I will do that on my end

Reply:
Reverted